packages/components/src/InputDate/InputDate.test.tsx

```diff
@@ -124,3 +127,62 @@ it("doesn't fire onChange when the new value is invalid", async () => {
   });
   expect(changeHandler).toHaveBeenCalledTimes(0);
 });
+
+it("doesn't display the calendar when input is focused with keyboard", () => {
+  const date = "11/11/2011";
+  const changeHandler = jest.fn();
+  const { queryByText, getByDisplayValue } = render(
+    <InputDate value={new Date(date)} onChange={changeHandler} />,
+  );
+  const form = getByDisplayValue(date);
+
+  act(() => {
+    fireEvent.focus(form);
+  });
+
+  expect(queryByText("15")).toBeNull();
```

Comment:
for consistency's sake and more descriptive

```suggestion
  expect(queryByText("15")).not.toBeInTheDocument();
```